app/controllers/chef_ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ChefIngredientsController < ApplicationController
+  def index
+    @chef = Chef.find(params[:id])
+    @ingredients = @chef.unique_ingredients
```

Comment:
Controller organization looks great! Keep in mind, too, that you can call @chef.unique_ingredients in the view as well so you don't need to create the second instance var necessarily